### PR TITLE
Split `rng_xfers` into its own crate

### PR DESCRIPTION
**Pull request by neysofu**
# Description
I'm splitting off `rng_xfers` into its own utility crate, to facilitate code reuse with the block explorer. 

## Testing
I'm mostly relying on existing tests, which covert pretty much all of this logic.

### CODE REVIEW

File: utils/rng-da-service/src/lib.rs [hunk 1]
```diff
@@ -15,81 +15,29 @@ use sov_rollup_interface::mocks::{
...
-
-fn generate_transfers(n: usize, start_nonce: u64) -> Vec<u8> {
-    let sender_address =
-        "sov15vspj48hpttzyvxu8kzq5klhvaczcpyxn6z6k0hwpwtzs4a6wkvqmlyjd6".to_string();
-    let token_name = "sov-test-token";
-    let sa = Address::from(
-        AddressBech32::try_from(sender_address)
-            .unwrap_or_else(|_e| panic!("Failed generating transfers")),
+pub fn sender_address_with_pkey() -> (Address, DefaultPrivateKey) {
+    let addr_bytes = "sov15vspj48hpttzyvxu8kzq5klhvaczcpyxn6z6k0hwpwtzs4a6wkvqmlyjd6".to_string();
```
Comment (vlopes11) [inline utils/rng-da-service/src/lib.rs:21]: Any reason why we don't generate the address with entropy (i.e. from a RNG)?

File: utils/rng-da-service/src/lib.rs [hunk 2]
```diff
@@ -15,81 +15,29 @@ use sov_rollup_interface::mocks::{
...
+    let addr = Address::from(
+        AddressBech32::try_from(addr_bytes)
```
Comment (vlopes11) [inline utils/rng-da-service/src/lib.rs:23]: If the origin string is constant, then the address could be

**Review by vlopes11 — APPROVED** [review]
LGTM; I pointed one question. But since it isn't introduced in this PR, we can merge as it is and maybe address the question @ future

**Review by citizen-stig — APPROVED** [review]

File: utils/rng-da-service/src/lib.rs (hunk 1 above)
Comment (neysofu) [reply utils/rng-da-service/src/lib.rs:21]: I don't think there's any particular reason; arguably we should. I'll make a note in the code.

Comment (neysofu) [reply utils/rng-da-service/src/lib.rs:21]: Left a comment! Thank you.

**Pull request merged**